packages/scatterbrain/src/abstract/render-server.ts

```diff
@@ -47,7 +48,7 @@ type Client = HTMLCanvasElement;
 export class RenderServer {
     private canvas: OffscreenCanvas;
     private refreshRequested: boolean;
-    regl: REGL.Regl | null;
```

Comment:
This change seems drastic, but we set `regl` in the `constructor` so it should be fine. I could go further and remove a bunch of `if(!regl)` sort of checks too but kept it simple for now